test/cloudant.test.js

```diff
@@ -68,6 +68,36 @@ describe('cloudant connector', function() {
           });
         };
       });
+    it('should create new instance if id provided', function(done) {
```

Comment:
You may use 'Primary key' instead of `id`; also you could add a test when the Primary key is not called `id` as well...